lib/std/mem.zig

```diff
@@ -602,32 +602,120 @@ test "span" {
     testing.expectEqual(@as(?[:0]u16, null), span(@as(?[*:0]u16, null)));
 }
 
-/// Same as `span`, except when there is both a sentinel and an array
-/// length or slice length, scans the memory for the sentinel value
-/// rather than using the length.
-pub fn spanZ(ptr: anytype) Span(@TypeOf(ptr)) {
```

Comment:
Did you intend to delete spanZ or just deprecate it?

Reply:
Thanks, I only intended to deprecate it, I got this right in the first round but appeared to have screwed it up in the revised version due to testing in a separate zig file to avoid running the entire stdlib test suite.